lib/daemon/index.js

```diff
@@ -188,18 +189,19 @@ function start(app, cb) {
 
 	var child = spawn(process.execPath, args, {
 		detached: true,
-		stdio: 'ignore'
+		stdio: ['ignore', 'ignore', 'pipe']
 	});
 
 	// assign the PID to the app
 
 	app.pid = child.pid;
+	app.stderr = child.stderr;
 
 	out(chalk.green.bold('Starting "' + appName + ' ' + args.join(' ') + '"... (pid: ' + child.pid + ')'));
 
 	// wait for a signal from the child process before we continue
 
-	sendSignal(null, null, timeouts.start, function (exitCode) {
+	sendSignal(app, null, timeouts.start, function (exitCode) {
 		// don't count the spawned app in the reference count of the daemonizer
 
 		child.unref();
```

Comment:
Does not seem like it's enough, game still dies violently soon after starting, tried to enable the syslog writer on top of the terminal one and I can't get any error message